tools/mtouch/mtouch.cs

```diff
@@ -878,6 +878,10 @@ public static bool CanWeSymlinkTheApplication (Application app)
 			if (app.EnvironmentVariables.Count > 0)
 				return false;
 
+			// If we are asked to run with concurrent sgen we also need to pass environment variables
```

Comment:
@rolfbjarne Is that comment right ?

If we pass environment variables to the sim app then the previous condition would disallow symlinking ? IOW this new condition would not be needed.

If we do **not** pass environment variables then disabling the fast-path is an option (that penalize build performance) as it will rebuild the launcher (and set the GC parameters). However the comment is incorrect / misleading.